guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>The link above is to generic and contain links to all the backbone/architectures available in the Hugging Face Transformer library. See if we can provide link that contains models for T5, Marian and Bart architectures.  See if we can use this link https://huggingface.co/models?filter=translation </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>UPDATED:</p><p> </p><p>Users can choose a supported backbone and a pretrained model based on the task the model is to be trained on. It is preferable to choose a model that was trained on a similar task and data.</p><p> </p><p>For instance, if a user is training a model for summarizing English text. It would be most appropriate to choose&nbsp;<code>T5</code>&nbsp;architecture and&nbsp;<code>t5-base-finetuned-summarize-news</code>&nbsp;model, as this model was trained for summarization downstream task.</p><p>One can search for available pretrained models in the&nbsp;<a href="https://huggingface.co/models" target="_blank">HuggingFace model zoo.</a></p>

Reply:
<p>please re-review the updated text once. Thanks!</p>

Reply:
<p>please re-review the updated text once. Thanks!</p>

Reply:
modified as per the discussion.